.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -58,4 +43,37 @@ body:
         ```
         -->
     validations:
-      required: true
\ No newline at end of file
+      required: true
+  - type: input
+    id: info-device
+    attributes:
+      label: Affected device
+      description: Which device isn't functioning as expected?
+      placeholder: e.g. Corsair Hydro H100i Pro XT
+    validations:
+      required: false
+  - type: input
+    id: info-os
+    attributes:
+      label: Operating system
+      description: Which OS are you using?
+      placeholder: e.g. Arch, macOS, Windows 10
+    validations:
+      required: true
+  - type: input
+    id: install-method
+    attributes:
+      label: Installation method
+      description: How did you install liquidctl? If the install failed please indicate what methods you tried.
+      placeholder: e.g. Official Arch Linux repository 
+    validations:
+      required: true
+  - type: input
+    id: info-version
+    attributes:
+      label: Version of liquidctl
+      description: Output of `liquidctl --version` If the install failed, indicate the version you tried to install.
+      placeholder: e.g. liquidctl v1.9.0 (Linux-5.16.16-arch1-1-x86_64-with-glibc2.35) 
+    validations:
+      required: true
```

Comment:
I couldn't think of a more concise way of putting this.

Reply:
It's fine!